src/eradiate/scenes/atmosphere/_core.py

```diff
@@ -129,6 +132,10 @@ def height(self) -> pint.Quantity:
         """
         return self.top - self.bottom
 
+    @property
+    def spectral_set(self) -> None | BinSet | WavelengthSet:
+        return None
```

Comment:
You're providing a default implementation, but you also reimplement this default in some child classes. It would be safer IMO to make this property abstract.

Reply:
Good point